docs/guides/managing-existing-services.md

```diff
@@ -93,14 +94,42 @@ Harmony validates that the image and tag are reachable - an error will be return
 **Important** from the [Docker documentation](https://docs.docker.com/engine/reference/commandline/image_tag/):
 >A tag name may contain lowercase and uppercase characters, digits, underscores, periods and dashes. A tag name may not start with a period or a dash and may contain a maximum of 128 characters.
 
+## Get backend service image tag update status
+
+You can get the status of backend service tag update by following the `status_link` returned in the backend service tag update response.
+
+For example:
+
+```
+curl -XGET -Ln -bj https://harmony.uat.earthdat.nasa.gov/service-image-tag/deployment/<deployment-id>
+```
+**Example 8** - Get service image tag update status
+
+The returned JSON response has the fields indicating the current status of the service image deployment.
+
+```JSON
+{
+  id: 1,
+  deployment_id: "befb50e0-e467-4776-86c8-e7218f1123cc",
+  username: "yliu10",
+  service: "giovanni-adapter",
+  tag: "new-version",
+  status: "successful",
+  message: null,
```

Comment:
I noticed the message is going to be null unless there is an issue. I think we should change the message to something generic like "Deployment in progress" when it first starts and "Deployment successful" when it is successful.